### PR TITLE
Fixed import of zip file

### PR DESCRIPTION
**Pull request by jhellar**
# Motivation
Fixes this issue https://issues.jboss.org/browse/RHMAP-880

# Changes
Use recommended approach to send form data.
(The previous approach should actually work as well according to documentation. https://github.com/request/request#multipartform-data-multipart-form-uploads)

### CODE REVIEW

File: lib/cmd/common/import.js [hunk 1]
```diff
@@ -93,7 +93,17 @@ function importAppNgui(projectId, appTitle, appTemplateType, fileOrRepo, deployE
...
+      title: appTitle,
+      templateType: appTemplateType,
+      templateZipFile: fs.createReadStream(file)
+    };
+
+    if (deployEnv) {
+      formData.autoDeployOption = deployEnv;
+    }
+
+    var r = request.post({url: url, headers:headers, formData: formData}, function(err, res, data) {
```
Comment (fh-qube) [inline lib/cmd/common/import.js]: MAJOR Remove the declaration of the unused 'r' variable. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnusedVariable)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by jasonmadigan** [comment]
@jhellar thanks for taking a look at this, reviewing now.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by jasonmadigan** [comment]
* Some SonarQube warnings
* Will want to bump the version in `package.json` & `sonar-project.properties`
* Can you provide some instructions on how to best verify the fix (sample commands etc.) I can grab a zip to import somewhere, but if you knew of one handy to grab that would be good too.

**Comment by jhellar** [comment]
Should be fixed.

**Comment by jasonmadigan** [comment]
Verification:

```
Zip archive: https://drive.google.com/open?id=0B59pU7uZgasXSGlTcDgtV0dLX1E
To import: `./bin/fhc.js import awofxs6movymoyk5t2zjs7jl test7 cloud_nodejs Archive.zip --env=qed1-mbaas2-dev`
the project is here: https://testing.zeta.feedhenry.com/#projects/awofxs6movymoyk5t2zjs7jl/apps
```

**Comment by jasonmadigan** [comment]
Verified that this change continues to see `fhc` work fine with zip archive import of apps. I also checked this on master, where it also works. 

@jhellar The original ticket mentioned that his only manifested as an issue when using `fh-fhc` as a library in something like UART - is there a way to easily check this too?

**Comment by jhellar** [comment]
@jasonmadigan it can be verified with this app: https://github.com/jhellar/zip-test.
Run: ./index.js -h <host> -u <username> -p <password> -e <env> -g <project-id> -t <app-title> -z <zip-archive>

**Comment by jasonmadigan** [comment]
@jhellar verified using your script which uses fhc as a library.

**Comment by jasonmadigan** [comment]
@jhellar this'll need a rebase before merging. Can set fixVersion on the JIRA to 3.15 too.

**Comment by jhellar** [comment]
Rebased.

**Comment by jasonmadigan** [comment]
@jhellar merged

**Pull request merged**